File: Basic Problem Solving/Palindrome-check/palindromeCheck.js
const palindromeCheck = num => {
  let dup = num
  let rev = 0
  while (num > 0) {
    let ld = num % 10
    rev = rev * 10 + ld
    num = Math.floor(num / 10)
  }
  console.log(rev)
  if (dup == rev) {
    return true
  } else {
    return false
  }
}

console.log(palindromeCheck(3773))
console.log(palindromeCheck(1234))

File: Basic Problem Solving/Palindrome-check/palindromeCheck.py
def palindromeCheck(num):
    dup = num
    rev = 0
    while num > 0:
        ld = num%10
        rev = rev * 10 + ld
        num = num//10
    
    if rev == dup:
        return True
    else:
        return False

print(palindromeCheck(3773))
print(palindromeCheck(1230))
